docs/src/main/paradox/scala/http/routing-dsl/directives/index.md

```diff
@@ -163,6 +163,12 @@ thing like this:
 
 @@snip [DirectiveExamplesSpec.scala](../../../../../../test/scala/docs/http/scaladsl/server/DirectiveExamplesSpec.scala) { #example-2 }
 
+### Composing Directives with `concat` Combinator
```

Comment:
Move it further down maybe? Now it looks the other combinators like `|` or `&` are under this new section